lib/workload/stateful/filemanager/filemanager/src/clients/aws/config.rs

```diff
@@ -0,0 +1,26 @@
+use aws_config::{defaults, BehaviorVersion, ConfigLoader, SdkConfig};
+use mockall::automock;
+
+/// A wrapper around a config loader.
+#[derive(Debug)]
+pub struct Config {
+    inner: ConfigLoader,
+}
+
+#[automock]
```

Comment:
This creates an entirely new `MockConfig` struct which has methods that allow setting expectations.